public/locales/fr.json

```diff
@@ -5,7 +5,7 @@
     "startPopulatingYourApp": "Prêt à partir? \nCommencez à remplir votre application avec des exemples de produits à afficher et à tester dans votre magasin.",
     "title": "Nom de l'application",
     "trophyAltText": "Beau travail sur la construction d'une application Shopify",
-    "yourAppIsReadyToExplore": "Votre application est prête à être explorée ! \nIl contient tout ce dont vous avez besoin pour démarrer, y compris le <polarisLink>système de conception Polaris</polarisLink>, l'<adminApiLink>API d'administration Shopify</adminApiLink> et la bibliothèque et les composants de l'interface utilisateur <appBridgeLink>App Bridge</appBridgeLink>."
+    "yourAppIsReadyToExplore": "Votre application est prête à être explorée ! \nElle contient tout ce dont vous avez besoin pour démarrer, y compris le <polarisLink>système de conception Polaris</polarisLink>, l'<adminApiLink>API d'administration Shopify</adminApiLink> et la bibliothèque et les composants de l'interface utilisateur <appBridgeLink>App Bridge</appBridgeLink>."
```

Comment:
May I suggest that we use comma and a final 'et' in enumerations, I believe this is a best practice in French:
```
le <polarisLink>système de conception Polaris</polarisLink>, l'<adminApiLink>API d'administration Shopify</adminApiLink> et la bibliothèque et les composants de l'interface utilisateur <appBridgeLink>App Bridge</appBridgeLink>."
```
would become
```
le <polarisLink>système de conception Polaris</polarisLink>, l'<adminApiLink>API d'administration Shopify</adminApiLink>, bibliothèque et les composants de l'interface utilisateur <appBridgeLink>App Bridge</appBridgeLink>."
```